hw/ip/pwrmgr/data/pwrmgr_testplan.hjson

```diff
@@ -7,7 +7,10 @@
   import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson"]
+                     "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson",
+                     // TODO: Top-level specific Hjson imported here. This will likely be resolved
+                     // once we move to IPgen flow.
+                     "hw/top_earlgrey/ip/pwrmgr/data/autogen/pwrmgr_sec_cm_testplan.hjson"]
```

Comment:
The pwrmgr_testplan.hjson file will need to become specific to top_earlgrey at some point.

Reply:
Should be cleaned up once pwrmgr is pulled into IPgen flow.